app/domain/authentication/authenticate.rb

```diff
@@ -86,6 +87,7 @@ def audit_failure(err)
           authenticator_name: authenticator_name,
           service: webservice,
           role: role,
+          client_ip: origin,
```

Comment:
It just occurs to me... is there a reason to have the two terms `client_ip` and `origin`?

Given `origin` is more vague, can we use `client_ip` everywhere?